src/Cheddar.js

```diff
@@ -1,3 +1,13 @@
 export { default as Controller } from './Controller'
 export { default as Model } from './Model'
 export { default as Router } from './Router'
+
+export class Cheddar {
+  static get database() {
+    return this._database
+  }
+
+  static set database(url) {
+    this._database = url
+  }
+}
```

Comment:
**Edit:** Don't test it, at least for now. I completely forgot [about this](http://en.wikipedia.org/wiki/Cyclomatic_complexity#Implications_for_software_testing). It's cyclomatic index is one, no need to test at all *for now*.